addons/mail/models/mail_mail.py

```diff
@@ -254,7 +254,7 @@ def send(self, auto_commit=False, raise_exception=False):
                 if raise_exception:
                     # To be consistent and backward compatible with mail_mail.send() raised
                     # exceptions, it is encapsulated into an Odoo MailDeliveryException
-                    raise MailDeliveryException(_('Unable to connect to SMTP Server'), exc)
+                    raise MailDeliveryException(_('Unable to connect to SMTP Server'))
```

Comment:
I don't think this change is needed here, I don't see any difference if the extra param is kept? Let me know why you did it, or keep minimal diff.